arangod/Cluster/ActionBase.cpp

```diff
@@ -171,9 +171,10 @@ void ActionBase::startStats() {
 
 /// @brief show progress on Action, and when that progress occurred
 void ActionBase::incStats() {
-  ++_progress;
+  // OSX clang cannot increment double very rare calls
+  std::lock_guard<std::mutex> guard(_progressMutex);
+  _progress = _progress + 1.0;
   _actionLastStat = secs_since_epoch();
```

Comment:
nit: `_actionLastStat` is now updated under the mutex, but read without it.
This is not racy, as `_actionLastStat` is an atomic variable. However, technically it is not necessary to update it under the mutex, and it is slightly confusing to readers of the code.

Reply:
Will fix.